tools/clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -13134,7 +13134,7 @@ void Sema::DiagnoseGloballyCoherentMismatch(const Expr *SrcExpr,
 
 void ValidateDispatchGridValues(DiagnosticsEngine &Diags,
                                 const AttributeList &A, Attr *declAttr) {
-  unsigned x = 1, y = 1, z = 1;
+  unsigned long long int x = 1, y = 1, z = 1;
```

Comment:
Should we use `uint64_t` instead?  I think that's more commonly used, isn't it?
I don't think it's necessary for the types of x,y,z to be changed, since this is the type retrieved.  Instead, we could use a `uint64_t product = (uint64_t)x * (uint64_t)y * (uint64_t)z;` later, right?

Reply:
Yes, that's logical.